### PR TITLE
Set `BBF_RUN_RARELY` for a scratch first BB if called count is 0.

### PR DESCRIPTION
**Pull request by pgavlin**
Just what it says on the tin.

Fixes VSO 469624.

### CODE REVIEW

**Comment by pgavlin** [comment]
@briansull @BruceForstall @dotnet/jit-contrib PTAL.

This depends on IBC data and JIT stress; I haven't been able to extract a self-contained repro case.

**Comment by briansull** [comment]
Does this only occur when the random BB Weight assigned a block via JitStress is zero?

File: src/jit/flowgraph.cpp [hunk 1]
```diff
@@ -12836,6 +12836,10 @@ void Compiler::fgComputeEdgeWeights()
         if (fgFirstBBisScratch())
         {
             fgFirstBB->setBBProfileWeight(fgCalledCount);
+            if (fgFirstBB->bbWeight == 0)
+            {
+                fgFirstBB->bbFlags |= BBF_RUN_RARELY;
```
Comment (briansull) [inline src/jit/flowgraph.cpp]: It may be better to do this check and set this flag in setBBProfileWeight();

**Comment by pgavlin** [comment]
x64 failure is a timeout in Chaos.

**Pull request merged**